spring-5-security-cognito/README.md

```diff
@@ -0,0 +1,3 @@
+## Relevant articles:
+
+- [Authenticating with Amazon Cognito Using Spring Security](http://www.baeldung.com/authenticating-with-amazon-cognito-using-spring-security)
```

Comment:
@adrianprecub thanks for adding this. Will you please make sure the link is `https`? Also, I think this will actually be called: `https://www.baeldung.com/spring-security-oauth-cognito`

Reply:
yep, i'll update the pr